src/pages/OfferNew/Checkout/index.tsx

```diff
@@ -204,34 +184,136 @@ export const Checkout: React.FC<Props> = ({
     }
   }, [isOpen])
 
+  const [signUiState, setSignUiState] = React.useState(SignUiState.NOT_STARTED)
+  const [bankIdUrl, setBankIdUrl] = React.useState<string | null>(null)
+  const [email, setEmail] = React.useState(firstQuote.email ?? '')
+  const [ssnUpdateLoading, setSsnUpdateLoading] = React.useState(false)
+  const [startPollingSignState, signStatus] = useSignState()
+  const [signQuotes, signQuotesMutation] = useSignQuotesMutation({
+    variables: { quoteIds: [firstQuote.id] },
+  })
+  const locale = useCurrentLocale()
+
+  const outerWrapper = React.useRef<HTMLDivElement>()
+
+  React.useEffect(() => {
+    if (
+      ![SignUiState.STARTED, SignUiState.STARTED_WITH_IFRAME].includes(
+        signUiState,
+      )
+    ) {
+      return
+    }
+
+    startPollingSignState()
+  }, [signUiState])
+
+  useTrack({
+    signState: signStatus?.signState,
+    email,
+    firstQuote,
+  })
+  useScrollLock(visibilityState, outerWrapper)
+
+  const canInitiateSign = Boolean(
+    signUiState !== SignUiState.STARTED &&
+      !signQuotesMutation.loading &&
+      emailValidation.isValidSync(email ?? '') &&
+      firstQuote.ssn,
```

Comment:
ssnValidation?

Reply:
I mean since this ssn comes from the backend I just assumed the ssn is correct, is that a false assumption?